src/components/Questionnaire/QuestionTypes/DateTimeQuestion.tsx

```
@@ -55,8 +55,11 @@ export function DateTimeQuestion({
   };
 
   const handleTimeChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const [hours, minutes] = event.target.value.split(":").map(Number);
-    if (isNaN(hours) || isNaN(minutes)) return;
+    const [hours, minutes] = event.target.value.split(":").map((value) => {
+      const number = Number(value);
+      return isNaN(number) ? undefined : number;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)